src/tribol/mesh/MfemData.hpp

```diff
@@ -780,8 +789,9 @@ class MfemMeshData
   /**
    * @brief Get the nodal response vector on the parent mesh
    *
-   * @param [out] r Pre-allocated, initialized mfem::Vector to which response
-   * vector is added
+   * @note This is stored as a dual vector, meaning the shared DOFs must be summed over all ranks to obtain their value.
```

Comment:
Consider: This is stored as a mfem dual vector, meaning that they expect the shared DOFs to be summed over all ranks to obtain their value.